src/components/AppSelect.tsx

```diff
@@ -0,0 +1,36 @@
+import { FormControl, InputLabel, MenuItem, Select, SelectChangeEvent } from '@mui/material'
+import { Value } from 'sass'
+
+interface Props {
+  labelGroup?: string
+  values: string[] | number[]
+  labels: string[]
```

Comment:
if we go with `options` prop we can use just `label` instead of `labelGroup`

if we don't, then let's use `groupLabel`